srcalc/src/main/java/gov/va/med/srcalc/vista/RpcVistaPatientDao.java

```diff
@@ -235,6 +268,38 @@ private void retrieveLabs(final int dfn, final Patient patient) throws ParseExce
         }
     }
     
+    private void retrieveHealthFactors(final int dfn, final Patient patient)
+    {
+        try
+        {
+            // Force a LinkedList to improve remove speed and allow for modification of the list
+            // since the doRpc uses Arrays.toList() which has a fixed size.
```

Comment:
Yuck, that's an implementation detail and doesn't belong here. The `doRpc()` Javadoc says the returned List is unmodifiable: that's the reason.